ballerina/googlesheets_client.bal

```diff
@@ -364,12 +364,18 @@ public isolated client class GoogleSheetsClient {
             return int:fromString(value.toString());
         } else if dataType == "time:Date" || dataType == "time:TimeOfDay" ||dataType == "time:Civil" || dataType == "time:Utc" {
             return self.stringToTime(value.toString(), dataType);
-        } else if dataType == "string" {
+        } else if dataType == "string"||dataType == "ENUM" {
             return value.toString();
         } else if dataType == "decimal" {
             return decimal:fromString(value.toString());
         } else if dataType == "float" {
             return float:fromString(value.toString());
+        } else if dataType == "boolean" {
+            if value.toString() == "TRUE" {
+                return true;
+            } else {
+                return false;
+            } 
```

Comment:
```suggestion
            return value.toString() == "TRUE";
```

Reply:
updated in [f7d8c93](https://github.com/ballerina-platform/module-ballerina-persist/pull/218/commits/f7d8c9364ffc8fa17d1d01c211a78e2af887b9bd)